metadata.json

```diff
@@ -25,7 +25,7 @@
     },
     {
       "name": "puppetlabs/powershell",
-      "version_requirement": ">= 1.0.0 <2.0.0"
+      "version_requirement": ">= 1.0.0 <3.0.0"
     },
     {
       "name": "liamjbennett/win_facts",
```

Comment:
@liamjbennett If you think it's worth it, would you be happy migrating win_facts to VP too?